docs/components/concepts/messages.md

```diff
@@ -4,7 +4,7 @@ title: "Messages"
 description: "Learn how process instances can respond to incoming messages."
 ---
 
-Process instances can respond to incoming messages. Published messages must be mapped onto a process instance. This step is called **[message correlation](/guides/message-correlation.md)**.
+Process instances can respond to incoming messages. Published messages must be mapped onto a process instance. This step is called message correlation.
```

Comment:
There are about 90 references of message correlation per version of docs -- perhaps we should create a backlog issue to add this term to the docs glossary and link to it if we are removing the doc on message correlation?

Reply:
Good idea, I am ok either doing that in the scope of this PR or a separate one.

Reply:
Would it make sense to add to the glossary, or link to message correlation [here](https://docs.camunda.io/docs/next/components/modeler/bpmn/message-events/#message-correlation)?

Reply:
I think linking to the existing documentation would be fine 👍